docs/topics/exporters.rst

```diff
@@ -167,9 +169,10 @@ BaseItemExporter
       value unchanged except for ``unicode`` values which are encoded to
       ``str`` using the encoding declared in the :attr:`encoding` attribute.
 
-      :param field: the field being serialized. If a raw dict is being
-          exported (not :class:`~.Item`) *field* value is an empty dict.
-      :type field: :class:`~scrapy.item.Field` object or an empty dict
+      :param field: the field being serialized. If the source :ref:`item object
+          <item-types>` does not define field metadata, *field* is an empty
+          :class:`dict`.
+      :type field: :class:`~scrapy.item.Field` object or an empty :class:`dict`
```

Comment:
```suggestion
      :type field: :class:`~scrapy.item.Field` object or a :class:`dict` instance
```

Reply:
the dict doesn't have to be empty, as we support meta for attr.s and dataclasses